php/Blockonomics.php

```diff
@@ -450,13 +452,7 @@ public function load_checkout_template($order_id, $crypto){
 
     // Load the the crypto options template in the page
     public function load_crypto_options_template(){
-        // Check to send the user to nojs page
-        if($this->is_nojs_active()){
-            $this->load_blockonomics_template('nojs_crypto_options');
-        }
-        else{
-            $this->load_blockonomics_template('crypto_options');
-        }
+        $this->load_blockonomics_template('crypto_options');
     }
```

Comment:
Quiz: Give two reasons why this function is not a good idea !

Reply:
Only used once and performs one function already covered. Removed in cbeaa28

Reply:
Yes it is unnecessarily creating an overhead of an extra function call and not providing and any modular improvement